src/components/form/MultiselectForm.tsx

```diff
@@ -52,13 +59,62 @@ class MultiSelectForm extends PureComponent<Props> {
                         type={'checkbox'}
                     />
                     <label htmlFor={option.name}>
-                        {option.name}
+                        {name}
                         <Amount>{option.count}</Amount>
                     </label>
                 </fieldset>
             </li>
         );
     }
+
+    constructor () {
+        super();
+
+        this.state = {
+            amountShown: MultiSelectForm.showStep
+        };
+
+        this.showMore = this.showMore.bind(this);
```

Comment:
Ik ben zelf meer een fan van een event handle functie definen als een fat arrow field aka
```ts
private showMore = () => {
...
```
aangezien die zichzelf al naar this binden